src/style/_markdown.scss

```diff
@@ -92,6 +92,9 @@
 
   img {
     max-width: 100%;
+    display: block;
```

Comment:
@feruzm why are we adding display and margins? i don't remember exactly but this attributes caused issues once.

Reply:
@talhasch Idea is to center images if they are not and margin auto works best with display block, you could try removing display block and see if it works as expected.

Reply:
this basically disallows users to add images aligned to left. removed styles. rest looks good.

Reply:
I did check pull-right and pull-left was working as expected and `<a>` style was about video embeds also looks better when they are centered. Centering images are really important if user doesn't use `<center>` or pull-right, pull-left then we should center it with styles.